web/api/alert.go

```diff
@@ -40,3 +46,4 @@ func (s AlertManagerService) addAlerts(w http.ResponseWriter, r *http.Request, p
 
 	s.Manager.Receive(alerts)
 }
+
```

Comment:
My earlier newline wording might have been confusing :) Unix text files should end with a single newline character (`0x0a`). This one ends with two now. Thus, there is now an empty line at the end (please remove it).